### PR TITLE
Add support for custom centroid range

### PR DESCRIPTION
**Pull request by eliasaxel**
Creates support for "wild centroids" for IDs between 35 000 – 40 000 as stated in the Helmet manual.

### CODE REVIEW

File: Scripts/datahandling/zonedata.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def __init__(self, data_dir: str, zone_numbers: numpy.array):
         parkdata = read_csv_file(data_dir, ".prk", self.zone_numbers, dtype)
         self.externalgrowth = read_csv_file(
             data_dir, ".ext",
-            all_zone_numbers[all_zone_numbers.searchsorted(external[0]):],
+            all_zone_numbers[all_zone_numbers.searchsorted(external[0]):all_zone_numbers.searchsorted(external[1],side='right')],
```
Comment (zptro) [inline Scripts/datahandling/zonedata.py:36]: This line is a bit long, but I guess this is otherwise the most concise way of doing this.

**Review by zptro — APPROVED** [review]
Nice fix!

**Review by hsl-petrhaj — APPROVED** [review]
Looks good, model run seems to work as well.

**Comment by hsl-petrhaj** [comment]
@eliasaxel You can now squash and merge. We have the habit that the author of the PR does it.

**Comment by hsl-petrhaj** [comment]
I merged, since Elias does not have the rights to do the merge. Thank you, Elias.

**Pull request merged**